src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
instead of using .properties use .yaml type so structure of configuration is more readable
 example below
<img width="425" alt="Bildschirmfoto 2021-07-08 um 14 12 55" src="https://user-images.githubusercontent.com/46439378/124919464-99c97e00-dff6-11eb-9a3e-3e6c00519d8a.png">

Reply:
- [ ] todo